### PR TITLE
Fix fetch in token server used in E2E Streaming Test

### LINKED ISSUE #2943: E2E Streaming Tests - "fetch is not defined"
## Versions
What package version of the SDK are you using. 4.10.x

## Describe the bug
Streaming tests bot is a node bot and uses "fetch" method to retrieve tokens. It was previously working without specifying a library to use fetch with, due to global pollution that has now been cleaned up:
https://github.com/microsoft/botbuilder-js/pull/2802/files#diff-805a496d3d37953468fa216291f2a5776d757dffbfaea6772f26a1ebc1fca956

Need to define fetch if none is available in global, otherwise, you won't get a DL token, if you can't make a request with a non-existent fetch.

## To Reproduce
Run [E2E_BF-Streaming-DL-ASE_Test](https://fuselabs.visualstudio.com/SDK_v4/_build?definitionId=1018) pipeline, which runs the offending bot

## Expected behavior
For pipeline to run successfully (i.e. for fetch to work and to receive a token)

### PR DESCRIPTION
**Pull request by Zerryth**
Fixes #2943 

## Description
Use `cross-fetch` to make token request.

## Testing
Created a [test pipeline](https://fuselabs.visualstudio.com/SDK_v4/_build/results?buildId=177666&view=results); runs the E2E DL ASE Streaming Tests successfully.

### CODE REVIEW

**Comment by joshgummersall** [comment]
[cross-fetch](https://github.com/lquixada/cross-fetch) handles the isomorphic piece so you can import and go. We should be moving to this package to keep our code more straightforward.

**Comment by Zerryth** [comment]
cross-fetch sounds good to me!

File: libraries/streaming-e2e-tests/bot/utils/getFetch.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
```
Comment (joshgummersall) [inline libraries/streaming-e2e-tests/bot/utils/getFetch.js]: This file can be removed now 😄

**Review by joshgummersall — APPROVED** [review]
Nice! Thanks 👍

**Review by stevengum — APPROVED** [review]

**Review by Zerryth — COMMENTED** [review]
um...can't seem to figure out how to get out of submit a review window..

**Pull request merged**